### ISSUE TITLE
Use currently saved settings on the login and other secure screens:  Option to ignore addons

### ISSUE THREAD

**Issue by robinettl** [nvaccess/nvda#10176]
When accessing the General page of the NVDA settings and choosing the option "Use currently saved settings on the login and other secure screens" option, a dialog box appears which reads "addons were detected in your user settings directory.  Copying these to the system profile could be a security risk.  Do you still wish to copy your settings?"  It seems I cannot mirror my current NVDA settings to secure screens without also copying my installed addons.  Could the user be given a third option on this screen instead of just yes or no, such as "continue without addons"?  I just want the same rate of speech, typing echo settings and so on, not necessarily all my plugins.

**Comment by josephsl** [comment]
Hi, rest assured that someone is indeed looking into this. CC @NVDAEs (for others, closing as duplicate of #6305)

**Comment by robinettl** [comment]
My mistake — didn’t see that existing ticket. Thanks.

**Comment by Adriani90** [comment]
Closing as duplicate
